<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>6</y>
      <width>371</width>
      <height>16</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>0</y>
      <width>98</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>选择文件</string>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>90</y>
      <width>192</width>
      <height>301</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="5" column="1">
      <widget class="QToolButton" name="toolButton_2">
       <property name="text">
        <string>查询最长字母数链</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>选择尾字母为</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>选择首字母为</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QToolButton" name="toolButton">
       <property name="text">
        <string>查询最长单词数链</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QToolButton" name="toolButton_3">
       <property name="text">
        <string>查询所有单词链</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="comboBox">
       <item>
        <property name="text">
         <string>None</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>a</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>b</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>c</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>d</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>e</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>f</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>g</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>h</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>i</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>j</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>k</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>l</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>m</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>n</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>o</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>p</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>q</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>r</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>s</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>t</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>u</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>v</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>w</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>x</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>y</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>z</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>是否含有环</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="comboBox_2">
       <item>
        <property name="text">
         <string>None</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>a</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>b</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>c</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>d</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>e</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>g</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>f</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>h</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>i</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>j</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>k</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>l</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>m</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>n</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>o</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>p</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>q</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>r</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>s</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>t</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>u</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>v</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>w</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>x</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>y</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>z</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="comboBox_5">
       <item>
        <property name="text">
         <string>否</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>是</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>40</y>
      <width>258</width>
      <height>444</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>输入文本为：</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QTextEdit" name="textEdit_2">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>查询结果为：</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QTextEdit" name="textEdit">
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'.AppleSystemUIFont'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
